doc/user/content/ops/troubleshooting.md

```diff
@@ -13,27 +13,17 @@ This page describes several SQL queries you can run to diagnose performance
 issues with Materialize.
 
 {{< warning >}}
-The introspection sources and views used in the queries below are subject to
-future changes without notice.
+The queries below reference [introspection relations](/sql/system-catalog/mz_internal/#replica-introspection-relations), which are subject to future changes without notice.
 
-This is why these sources and views are all created in the `mz_internal` schema,
+This is why these relations are all created in the `mz_internal` schema,
 and why it is not possible to create higher-level views depending on them.
 {{< /warning >}}
 
 ## Limitations
 
-Introspection sources are maintained by independently collecting internal logging
-information within each of the replicas of a cluster. Thus, in a multi-replica
-cluster, the queries below need to be directed to a specific replica by issuing
-the command `SET cluster_replica = <replica_name>`. Note that once this command
-is issued, all subsequent `SELECT` queries, for introspection sources or not, will
-be directed to the targeted replica. The latter selection can be cancelled by
-issuing the command `RESET cluster_replica`.
-
-As a consequence of the above, you should expect the answers to the queries below
-to vary dependending on which cluster you are working in. In particular, indexes
-and dataflows are local to a cluster, so their introspection information will
-vary across clusters.
+The contents of introspection relations differ depending on the selected cluster and replica.
+As a consequence, you should expect the answers to the queries below to vary depending on which cluster you are working in.
+In particular, indexes and dataflows are local to a cluster, so their introspection information will vary across clusters.
```

Comment:
Most of what was previously here now lives in the `mz_internal` docs for introspection sources, as I think it should. It's probably still a good idea to point out the most important pitfalls here, as users wanting to troubleshoot something might not think to click through to the `mz_internal` docs.

Reply:
@teskje Could you point out a "top" pitfall and we can document it here?